src/producer.rs

```diff
@@ -285,11 +285,14 @@ impl Archive {
                     panic!("Failed to create a symlink {:?} -> {:?}", src_path, path)
                 });
 
+                // On Windows, creating a symbolic link to a file requires Administrator
+                // rights. Fall back to copying if creating the symbolic link fails.
                 #[cfg(windows)]
-                os::windows::fs::symlink_file(&src_path, path).unwrap_or_else(|_| {
-                    panic!("Failed to create a symlink {:?} -> {:?}", src_path, path)
-                });
-
+                os::windows::fs::symlink_file(&src_path, &path)
+                    .or_else(|_| std::fs::copy(&src_path, &path).map(|_| ()))
+                    .unwrap_or_else(|_| {
+                        panic!("Failed to create a symlink {:?} -> {:?}", src_path, path)
```

Comment:
Could you update the error message?

Reply:
Sure. What would you like it to be?  Something like `"Failed to create symlink / copy file"` or `"Failed to copy file"`, or something else?

Reply:
If we implement the symlink creation warning I suggested below, we can say "Failed to copy file".
If we don't implement the warning, we can say "Failed to create symlink or copy file".

Reply:
It's a bit hard... with big projects it can become awful.
I wonder if should use an option to allow that...
If some people are using grcov with a kind of big project and they aren't aware about that stuff then each time it'll run a forest will burn. And if we've an option (`--copy-when-no-symlink` or whatever) then the first run will fail because the option isn't used, and then with a good error message they can decide to copy or to allow symlink creations in configuring windows correctly.